molecule/molecule_json_loader.h

```diff
@@ -43,13 +48,26 @@ namespace indigo
     {
     public:
         DECL_ERROR;
+        explicit MoleculeJsonLoader( rapidjson::Value& molecule, rapidjson::Value& rgroups );
```

Comment:
Are you sure that constructor with two rapidjson semi-parsed structures is a handy and enough-abstract way of creating JsonLoader?
I'd load class internals with parsing job, leaving:
```
explicit MoleculeJsonLoader( const char * json_data);
```
Even wasting the previous parsing done by caller. 
BTW and IMHO, there is dangerous operation by re-making those `molecule` and `rgroups` nodes.

Reply:
Yeah, treat this as optional proposal. Can be improved in the future.

Reply:
It has been done intentionally because KET-format had multiple molecules in the previous version and common r-groups list.